src/pages/EarnTri/Manage.tsx

```diff
@@ -195,9 +195,15 @@ export default function Manage({
         <PoolData>
           <AutoColumn gap="sm">
             <TYPE.subHeader>{t('earnPage.poolRate')}</TYPE.subHeader>
+             {noTriRewards ? (
             <TYPE.body fontSize={24} fontWeight={500}>
-              {`${totalRewardRateFriendly}` + t('earnPage.pngPerWeek')}
+              {'Non TRI Pool'}
```

Comment:
nit strings here aren't necessary
```suggestion
              Non TRI Pool
```